save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -112,7 +141,53 @@ external interface OrganizationContestsTableProps<D : Any> : TableProps<D> {
 private fun organizationContestsMenu() = FC<OrganizationContestsMenuProps> { props ->
     val (isToUpdateTable, setIsToUpdateTable) = useState(false)
     val (isContestCreationModalOpen, setIsContestCreationModalOpen) = useState(false)
+    val (isDeleteContestModalOpen, setDeleteContestModalOpen) = useState(false)
+    val (deletingContest, setDeletingContest) = useState<ContestDto?>(null)
+    val (confirmLabel, setConfirmLabel) = useState("")
+    val (confirmMessage, setConfirmMessage) = useState("")
+    val (contests, setContests) = useState<MutableSet<ContestDto>>(mutableSetOf())
     val refreshTable = { setIsToUpdateTable { !it } }
+    val deleteCont: (ContestDto) -> Unit = { contest ->
+        contests.remove(contest)
+        setContests(contests)
```

Comment:
It's preferrable to use immutable collections for states, because it will prevent you from accidental update of collection without calling state setter. Immutable collections can return a copy without a removed element

Reply:
Fixed

Reply:
There's still calls to `.toMutableSet()` when fetching contests. Can be replaced with `.toSet()`

Reply:
Fixed